ap/tasks/selection.py

```diff
@@ -46,25 +46,32 @@ def requires(self):
         return reqs
 
     def output(self):
-        return {
+        outputs = {
             "res": self.local_target(f"results_{self.branch}.parquet"),
             "stat": self.local_target(f"stats_{self.branch}.json"),
         }
```

Comment:
see comment in file `ap/tasks/reduction.py` regarding descriptive power of keys of dictionary. Wouldn't it be better to make it clearer?
```suggestion
        outputs = {
            "result": self.local_target(f"results_{self.branch}.parquet"),
            "stats": self.local_target(f"stats_{self.branch}.json"),
        }

Reply:
Yep, added that ✓